app/frontend/javascripts/classes/StoreManagerMixin.js

```diff
@@ -333,6 +334,15 @@ export const mixin = {
           }
         })
         .catch((e) => {
+          const body = document.querySelector('body');
+          const modalWindow = new ModalWindow(body, 'Some error!');
+          modalWindow.show();
+
+          body.addEventListener('click', (e) => {
+            if (e.target !== modalWindow.messageArea) {
+              modalWindow.hide();
+            }
+          });
```

Comment:
基本的にこれでいいと思いますが、
今思いましたのは、catch が動作されたたびに `body` に　新しい `eventListener` が追加されていくので、メモリーリークになります。。
ここで、Modal を解除したら、その eventListener をも削除しといたほうが安全：
```suggestion
const clickHandler = (e) => {
    if (e.target !== modalWindow.messageArea) {
        modalWindow.hide();
        body.removeEventListener("click", clickHandler);
    }
};
body.addEventListener("click", clickHandler);

```